src/bin/common/parse_command.ml

```diff
@@ -910,6 +911,23 @@ let parse_output_opt =
       Arg.(value & flag & info ["dump-models"] ~doc ~docs:s_models)
     in
 
+    let dump_models_on =
+      let doc =
+        "Select a channel to output the models."
```

Comment:
This doc should make it clear that the option only applies to `--dump-model`

Reply:
Done!